MediaBrowser.Controller/Entities/TV/Series.cs

```diff
@@ -66,6 +66,8 @@ public Series()
         /// </remarks>
         public string DisplayOrder { get; set; }
 
+        public string EpisodeGroupId { get; set; }
```

Comment:
Another nitpick:
Please add summary remarks for consistency

Reply:
Done!